### PR TITLE
Feat: navigation component

### PR DESCRIPTION
**Pull request by Muzammil-khan710**
Added navigation Component and updated links

### CODE REVIEW

File: docs/components/Navigation/navigation.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/styles.css">
+    <title>Modal</title>
+</head>
+<body>
```
Comment (Shubhamkr2610) [inline docs/components/Navigation/navigation.html]: I think name of title should be changed to  navigation

**Review by Shubhamkr2610 — APPROVED** [review]
your code is clean and readable , easy to understand . You are good to go

File: docs/components/Navigation/navigation.html (hunk 1 above)
Comment (Muzammil-khan710) [reply docs/components/Navigation/navigation.html]: Thank you

**Review by sanu29 — APPROVED** [review]
Good to merge

**Review by srishtigautam21 — APPROVED** [review]
The UI looks great
No changes from my side
Good work Good to merge

**Pull request merged**